src/llama_stack/apis/agents/openai_responses.py

```diff
@@ -495,6 +496,9 @@ class OpenAIResponseInputToolMCP(BaseModel):
     server_label: str
     server_url: str
     headers: dict[str, Any] | None = None
+    # OAuth access token for MCP server authentication
```

Comment:
don't add comments here which are not represented above in the comment block since this is part of the API